### PR TITLE
Add operator manifests extraction information

### PR DESCRIPTION
**Pull request by athos-ribeiro**
* OSBS-6686

Signed-off-by: Athos Ribeiro <athos@redhat.com>

### CODE REVIEW

File: docs/users.rst [hunk 1]
```diff
@@ -625,3 +625,37 @@ same effect, specify such images with another stage, for example::
     COPY --from=source1 src/ dest/
 
 
+Operator manifests
+------------------
+
+An operator_ is an image with additional metadata. This image should contain a
```
Comment (MartinBasti) [inline docs/users.rst]: `An operator_ is an image with additional metadata.` Please drop this sentence or use something from https://coreos.com/operators/

Comment (MartinBasti) [reply docs/users.rst]: Maybe `OSBS is able to extract operator manifests from an operator image.`   [operator image] can be linked to https://coreos.com/operators/

Comment (athos-ribeiro) [reply docs/users.rst]: Done

**Comment by MartinBasti** [comment]
Bump, AFAIK atomic-rector with this feature has been released, so we can merge this anytime we got second approval.

**Pull request merged**